components/ai_chat/resources/page/ai_chat_ui.html

```diff
@@ -29,7 +29,7 @@
     </style>
   </head>
   <body>
-    <script type="module" src="chat_ui.bundle.js"></script>
     <div id="mountPoint"></div>
+    <script src="chat_ui.bundle.js"></script>
```

Comment:
could we keep the type here as `module` and add the flag to the BUILD.gn to output a module? I think we should be gradually migrating anyway

Reply:
Should the main bundle be module? The reason I removed it was to have it load as eagerly as possible, but maybe I'm mistaken?

Reply:
I think in this scenario, yes. So `<script type=module>` is `defer` by default so it will execute after the HTML has finished parsing (but before `DOMContentLoaded` fires), but it will start loading as soon as the parser sees the tag in the document.

I think we should put the `<script type=module src=..>` in the head of the document. Using the raw script tag will block further parsing of the HTML file until parsing of the script completes (though this is at the end of the document so its probably fine).

Either way, I don't think there should be a huge difference.

https://jakearchibald.com/2017/es-modules-in-browsers/